src/main/resources/db/changelog/20220314130000-clear_product_entity_versions.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
```

Comment:
We might want to make it a habit to start using the `4.8.xsd` version since if we don't then liquibase will complain if we try to use newer XML tag features than are available in 3.1 (see [here](https://docs.liquibase.com/concepts/changelogs/xml-format.html)). But this is just a note for future reference (not change needed here).